app/models/identifier.rb

```diff
@@ -27,21 +27,27 @@ class Identifier < ActiveRecord::Base
 
   belongs_to :identifiable, polymorphic: true
 
-  belongs_to :identifier_scheme
+  # TODO: uncomment 'optional: true' once we are on Rails 5
+  belongs_to :identifier_scheme #, optional: true
```

Comment:
Thanks for adding these forward-looking TODO's

Reply:
👍 we should run a search after the upgrade and address them (or remove if they are no longer relevant)